state/querytracker.go

```diff
@@ -93,10 +95,13 @@ func (q *queryTracker) TrackRead(collectionName string, query interface{}) {
 	q.mu.Lock()
 	defer q.mu.Unlock()
 
-	q.queries = append(q.queries, QueryDetails{
-		Type:           "read",
-		CollectionName: collectionName,
-		Query:          query,
-		Traceback:      string(debug.Stack()),
-	})
+	traceback := string(debug.Stack())
+	if strings.Contains(traceback, q.method) {
```

Comment:
Just say the trackback is for FullStatusFoo, it will match I think.
Can we add a delimiter to the q.method string to tighten the match.